ui/components/component-library/modal-body/README.mdx

```diff
@@ -0,0 +1,80 @@
+import { Controls, Canvas } from '@storybook/blocks';
+
+import * as ModalBodyStories from './modal-body.stories';
+
+# ModalBody
+
+`ModalBody` is a container component that handles the scrolling of the `Modal` content.
+
+<Canvas of={ModalBodyStories.DefaultStory} />
+
+## Props
+
+The `ModalBody` accepts all props below as well as all [Box](/docs/components-ui-box--default-story#props) component props
```

Comment:
Remove this line no longer needed
```suggestion
```

Reply:
Updated https://github.com/MetaMask/metamask-extension/pull/22154/commits/8e21ac564895f55f2eb69bfa7644db9d787e266c